herzog/parser.py

```diff
@@ -26,16 +26,16 @@ def __init__(self, cell_type: CellType, lines: Iterable[str]):
                 else:
                     self.lines.append(line)
             elif CellType.markdown == self.cell_type:
-                if '"""' == line:
+                if line in ('"""', "pass"):
```

Comment:
Neither `"""` nor `pass` are a part of the markdown proper.  This is to keep `pass` from being duplicated each time (since it must be added, as it's not a part of the ipython notebook to begin with, especially one downloaded from Terra).

I'm not sure that the change makes sense outside of the context of this PR, but erring on the side of caution, I will make a new PR.